lib/maplibre-compose/src/commonMain/kotlin/org/maplibre/compose/location/GeoLocator.kt

```diff
@@ -0,0 +1,39 @@
+package org.maplibre.compose.location
+
+import androidx.compose.runtime.Composable
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.flow.StateFlow
+
+public interface GeoLocator {
+  public val location: StateFlow<Location?>
+}
+
+public enum class DesiredAccuracy {
+  Highest,
+  High,
+  Balanced,
+  Low,
+  Lowest,
+}
```

Comment:
would be good to add some documentation comments indicating how we map these to platform accuracies